generator/altitude_generator.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "std/string.hpp"
+
+namespace routing
+{
+void BuildRoadAltitudes(string const & srtmPath, string const & baseDir,
```

Comment:
Why do you need baseDir? Why not to directly pass full path to the mwm? Also, note that for better testing all what you need is to pass a vector of pairs ```(featureId, vector of feature points)```. I suggest you to change the signature to:
```c++
void SerializeAltitudes(Writer & writer, SrtmTileManager & manager, vector<pair<uint32_t, vector<m2::PointD>>> & roads);
```